upstairs/src/test.rs

```diff
@@ -5093,6 +4975,7 @@ pub(crate) mod up_test {
     }
 
     #[tokio::test]
+    #[should_panic]
```

Comment:
If we should panic here, then we unwrap on `encrypt_in_placed` won't this give us a false negative?

Reply:
You're right, see 707f4d9 where I rolled this back